### ISSUE TITLE
Add Part_ID to Parts>>Containers Batch Tool

### ISSUE THREAD

**Issue by Nicole-Ridgwell-NMMNHS** [ArctosDB/arctos#4139]
**Is your feature request related to a problem? Please describe.**
I'm in the process of creating object tracking from an inventory. I need to be able add parts by part_id otherwise I have to manually add any specimen with duplicate part names. This is even more critical now that we've simplified parts with the part modifiers attribute.

**Describe what you're trying to accomplish** 
Loading parts to containers with a unique ID.

**Describe the solution you'd like**
Add Part_ID to the Parts>>Containers Batch Tool. I realize Part_ID isn't stable, but for this purpose, I don't think that is necessary. 

**Describe alternatives you've considered**
Individually updating the container for duplicate parts. This will become extremely difficult when I get to records with a lot of duplicate parts.

**Comment by Jegelewicz** [comment]
This functionality has been needed for quite some time!

**Comment by dustymc** [comment]
@acdoll here are some old data that I had to move to rebuild the table

[cf_temp_barcode_parts.csv.zip](https://github.com/ArctosDB/arctos/files/7929912/cf_temp_barcode_parts.csv.zip)
.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Yay, thank you!

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Just uploaded a set with part id, had no problem!

**Comment by dustymc** [comment]
Thanks for the feedback; it's good to know what's actually getting used!

**Comment by acdoll** [comment]
I didn't know those were in there - thanks for saving them @dustymc . I'm trying to remember exactly what I was doing here - where can I search for the container IDs?  I thought that would be in the OID search in Object Tracking but that isn't finding any of this (even the ones that I can verify the collection_object_id from the catalog record).

**Comment by campmlc** [comment]
Yes, could we make these more available? I can see them in catalog record parts edit page, and in parts table download from search results. It would be helpful to have them in Review Loan Items too - with a way to assign a PID as a permanent identifier?

**Comment by dustymc** [comment]
@acdoll parts table download is probably the best place, it's denormalized to parts.

>Review Loan Items

Possible - as is anything with parts as "rows" - needs an Issue.

>permanent

That's https://github.com/ArctosDB/arctos/issues/3630, which looks to be abandoned from here.

Maybe @Nicole-Ridgwell-NMMNHS could be talked into a how-to?

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
This needs to be updated: [https://handbook.arctosdb.org/how_to/How-to-Bulkload-Barcodes-to-Specimen-Parts.html](https://handbook.arctosdb.org/how_to/How-to-Bulkload-Barcodes-to-Specimen-Parts.html)

And parts>>containers should probably also be discussed here: [https://handbook.arctosdb.org/how_to/How-to-Start-Object-Tracking.html#placing-parts-in-containers](https://handbook.arctosdb.org/how_to/How-to-Start-Object-Tracking.html#placing-parts-in-containers)

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Not a how-to exactly, but here is my workflow that shows how I use this tool: https://github.com/ArctosDB/Arctos-Workflows/blob/main/.github/ISSUE_TEMPLATE/NMMNH%20Inventory%20Upload.md
